modules/hal_nordic/nrf_802154/sl_opensource/platform/nrf_802154_clock_zephyr.c

```diff
@@ -63,53 +74,27 @@ void nrf_802154_clock_hfclk_stop(void)
 	hfclk_is_running = false;
 }
 
-bool nrf_802154_clock_hfclk_is_running(void)
-{
-	return hfclk_is_running;
-}
+#elif defined(NRF54H_SERIES)
 
-static void lfclk_on_callback(struct onoff_manager *mgr,
-			      struct onoff_client  *cli,
-			      uint32_t state,
-			      int res)
-{
-	lfclk_is_running = true;
-	nrf_802154_clock_lfclk_ready();
-}
+#define NRF_LRCCONF_RADIO_PD NRF_LRCCONF010
 
-void nrf_802154_clock_lfclk_start(void)
+void nrf_802154_clock_hfclk_start(void)
 {
-	int ret;
-	struct onoff_manager *mgr =
-		z_nrf_clock_control_get_onoff(CLOCK_CONTROL_NRF_SUBSYS_LF);
-
-	__ASSERT_NO_MSG(mgr != NULL);
+	/* Use register directly, there is no support for that task in nrf_lrcconf_task_trigger */
+	NRF_LRCCONF_RADIO_PD->EVENTS_HFXOSTARTED = 0x0;
+	NRF_LRCCONF_RADIO_PD->TASKS_REQHFXO = 0x1;
 
-	sys_notify_init_callback(&lfclk_cli.notify, lfclk_on_callback);
-
-	ret = onoff_request(mgr, &lfclk_cli);
-	__ASSERT_NO_MSG(ret >= 0);
+	hfclk_is_running = true;
+	nrf_802154_clock_hfclk_ready();
 }
 
-void nrf_802154_clock_lfclk_stop(void)
+void nrf_802154_clock_hfclk_stop(void)
 {
-	int ret;
-	struct onoff_manager *mgr =
-		z_nrf_clock_control_get_onoff(CLOCK_CONTROL_NRF_SUBSYS_LF);
+	/* Use register directly, there is no support for that task in nrf_lrcconf_task_trigger */
+	NRF_LRCCONF_RADIO_PD->TASKS_STOPREQHFXO=0x1;
```

Comment:
To satisfy compliance checks.
```suggestion
	NRF_LRCCONF_RADIO_PD->TASKS_STOPREQHFXO = 0x1;
```